composer.json

```diff
@@ -12,7 +12,8 @@
         "grpc/grpc": "^1.4",
         "google/protobuf": "^3.5.1",
```

Comment:
I noticed [here](https://github.com/googleapis/google-cloud-php/pull/1517/files#diff-ad6978f3626f02acfb16dbe103da2356R9) we are requiring a higher version of protobuf, would there be any harm in doing the same here?

Reply:
SGTM, done